src/extension/helpers.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
```

Comment:
when creating a new file we shoudl use the curren tyear

Reply:
lol.. catched by copy/paste monster 🙁 😃

Reply:
fixed